src/test/java/io/appform/dropwizard/actors/connectivity/actor/RMQTest.java

```diff
@@ -0,0 +1,96 @@
+package io.appform.dropwizard.actors.connectivity.actor;
+
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.health.HealthCheckRegistry;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.GetResponse;
+import io.appform.dropwizard.actors.ConnectionRegistry;
+import io.appform.dropwizard.actors.TtlConfig;
+import io.appform.dropwizard.actors.actor.ActorConfig;
+import io.appform.dropwizard.actors.utils.ActorType;
+import io.appform.dropwizard.actors.utils.AsyncOperationHelper;
+import io.appform.dropwizard.actors.utils.EmbeddedRMQServer;
+import io.appform.dropwizard.actors.utils.SidelineTestActor;
+import io.appform.dropwizard.actors.utils.TestMessage;
+import io.appform.dropwizard.actors.config.RMQConfig;
+import io.appform.dropwizard.actors.connectivity.RMQConnection;
+import io.appform.dropwizard.actors.exceptionhandler.ExceptionHandlingFactory;
+import io.appform.dropwizard.actors.retry.RetryStrategyFactory;
+import io.dropwizard.Configuration;
+import io.dropwizard.jackson.Jackson;
+import io.dropwizard.setup.Environment;
+import lombok.SneakyThrows;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import javax.validation.Validation;
+import java.util.concurrent.Executors;
+
+public class RMQTest {
+    private static ConnectionRegistry connectionRegistry;
```

Comment:
Refer `NamespacedQueuesTest` for mode of writing tests

Reply:
Updated test in `NamespacedQueuesTest` and removed RMQ Test.